infrasim/console.py

```diff
@@ -170,11 +175,16 @@ def stop(instance="node-0"):
         pass
 
 
-def console_main(instance="node-0"):
+def console_main(instance="default"):
     if len(sys.argv) < 2:
-        print "ipmi-console [ start | stop ]"
+        print "ipmi-console [ start | stop ] ( node_name )"
```

Comment:
@WinnieXU1 , I would suggest you use this bracket `[]` as a convention.
You can have a lot of examples by see their help or manual:

    guf1@ubuntu:~$ ssh -h
    unknown option -- h
    usage: ssh [-1246AaCfgKkMNnqsTtVvXxYy] [-b bind_address] [-c cipher_spec]
               [-D [bind_address:]port] [-E log_file] [-e escape_char]
               [-F configfile] [-I pkcs11] [-i identity_file]
               [-L [bind_address:]port:host:hostport] [-l login_name] [-m mac_spec]
               [-O ctl_cmd] [-o option] [-p port]
               [-Q cipher | cipher-auth | mac | kex | key]
               [-R [bind_address:]port:host:hostport] [-S ctl_path] [-W host:port]
               [-w local_tun[:remote_tun]] [user@]hostname [command]
    guf1@ubuntu:~$